docs/source/install/installation.rst

```diff
@@ -294,144 +294,21 @@ If you open a new terminal for example, be sure to activate it first with::
 At this point, you can choose to read on for additional installation details and configuration options, or you can choose to start using
 AiiDA and go straight to the section :ref:`get started<get_started>`.
```

Comment:
This should be changed, because now the other sections assume that the daemon is started and running. But following this line people might skip it. Either put this line after the start the daemon section or return the start daemon section to one of the points in `get_started`

Reply:
> Either put this line after the start the daemon section 

thanks, will do!